packages/lib-classifier/src/components/Classifier/ClassifierContainer.js

```diff
@@ -81,8 +81,17 @@ async function fetchWorkflow(workflowID) {
 }
 
 function useWorkflowSnapshot(workflowID) {
-  const { data } = useSWR(workflowID, fetchWorkflow)
-  return data ?? null
+  const [prevSnapshot, setPrevSnapshot] = useState(null)
+  const response = useSWR(workflowID, fetchWorkflow)
+  let data = null
+  if (workflowID) {
+    data = response.data
+  }
+  if (data && prevSnapshot?.id !== data.id) {
+    setPrevSnapshot(data)
+    return data
+  }
+  return prevSnapshot
```

Comment:
This beefs up `useWorkflowSnapshot` so that it returns the last workflow when `workflowID` is undefined.